.circleci/config.yml

```diff
@@ -280,7 +280,12 @@ step-maybe-electron-dist-strip: &step-maybe-electron-dist-strip
     command: |
       if [ "$STRIP_BINARIES" == "true" ] && [ "`uname`" != "Darwin" ]; then
         cd src
-        electron/script/strip-binaries.py --target-cpu="$TARGET_ARCH"
+        mkdir out/Default/debug
+        # Generate electron.debug. Required to do it from there in order to maintain debug references.
+        cd out/Default/debug
+        ../../../electron/script/copy-debug-symbols.py --target-cpu="$TARGET_ARCH" --compress
+        ../../../electron/script/strip-binaries.py --target-cpu="$TARGET_ARCH"
+        ../../../electron/script/copy-debug-symbols.py --target-cpu="$TARGET_ARCH" --add-debug-link
```

Comment:
I'd prefer two separate scripts for two separate tasks, rather than using a flag on the script to control behavior.

Reply:
Done